include/f_op/f_op_actor_mng.h

```diff
@@ -337,16 +337,16 @@ inline void dComIfGs_offSwitch(int i_no, int i_roomNo);
 inline BOOL dComIfGs_isSwitch(s32 i_no, s32 i_roomNo);
 inline void dComIfGs_offActor(int i_no, int i_roomNo);
 
-inline void i_fopAcM_onSwitch(const fopAc_ac_c* pActor, int sw) {
-    return dComIfGs_onSwitch(sw, fopAcM_GetHomeRoomNo(pActor));
+inline void fopAcM_onSwitch(const fopAc_ac_c* pActor, int sw) {
+    return dComIfGs_onSwitch(sw, pActor->orig.roomNo);
```

Comment:
are these changes needed to match? tp debug at least confirms `fopAcM_GetHomeRoomNo` was used

Reply:
It's needed for the regalloc to match, not only in obj_movebox but also in item and andsw0.

Although `fopAcM_GetHomeRoomNo` does still appear in the debug symbol maps for obj_movebox and andsw0, which I guess indicates it is used? Maybe there's a better fix.

Reply:
maybe just add a comment about it for now so it can be revisited if the issue comes back the other way around where it only matches with the inline